package.json

```diff
@@ -842,7 +828,6 @@
     "url-loader": "^2.2.0",
     "val-loader": "^1.1.1",
     "vinyl-fs": "^3.0.3",
-    "wait-on": "^5.2.1",
```

Comment:
This is used here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/e2e/run-e2e.sh#L23

I am sure we could replace it's usage with our own tools, though.

Reply:
Good catch - pushed with 8e8fe55.